core/components/molecules/tabs/Tabs.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import classNames from 'classnames';
 import { Pills, Icon, Text, Tab } from '@/index';
 import { BaseProps, extractBaseProps, SingleOrArray } from '@/utils/types';
+import { moveToIndex } from '../../organisms/grid/utility';
```

Comment:
Please do not use the utility function of another component. Either create a common utility file for all the component or create a utility function specific to that component